places-rest-apis/adobe-i-o-integration/README.md

```diff
@@ -66,23 +65,35 @@ The following video guides you through the process of generating the key pair:
 To create a Places integration:
 
 1. Go to [https://console.adobe.io/](https://console.adobe.io/) and sign in with your Adobe ID.
-2. If you have access to more than one Experience Cloud organization, select the organization from the drop-down list on the left.
-3. Click **New Integration**.
-4. Select **Access an API** and click **Continue**.
-5. Under **Experience Cloud**, select **Places** as the Adobe service to which you want to integrate and click **Continue**.
+
+2. Click on the **Create integration** in the **Quick Start** section.
+
+3. Select **Access an API** (default selection) and click **Continue**.
+
+4. If you have access to more than one Experience Cloud organization, select the desired organization from the drop-down list on the top right.
+
+5. Under **Experience Cloud**, select **Places** as the Adobe service to which you want to integrate with and click **Continue** near the bottom of the page.
+
 6. Select **New integration** and click **Continue**.
 7. On the _Create a new integration_ screen, enter a name and description.
 8. Drag and drop your `xxxx_public.crt` file, created above, to the **Public keys certificates** drop zone.
-9. At the bottom of the page, click **Create integration**.
-10. After a few seconds, in the _Integration created_ screen, verify that the following message appears:
+
+9. Select a product profile. Check with System Admin for you organization if unsure which to select.
```

Comment:
Please restructure as follows:
"9. Select a product profile. 

  If you are unsure of which profile to select, contact your system administrator.